pkg/logs/sds/scanner.go

```diff
@@ -102,24 +103,28 @@ func (s *Scanner) Reconfigure(order ReconfigureOrder) error {
 	case StandardRules:
 		// reconfigure the standard rules
 		err := s.reconfigureStandardRules(order.Config)
+		var isActive bool
 
 		// if we already received a configuration and no errors happened while
 		// reconfiguring the standard rules: reapply the user configuration now.
 		if err == nil && s.rawConfig != nil {
-			if rerr := s.reconfigureRules(s.rawConfig); rerr != nil {
+			var rerr error
+			if isActive, rerr = s.reconfigureRules(s.rawConfig); rerr != nil {
 				log.Error("Can't reconfigure SDS after having received standard rules:", rerr)
```

Comment:
```suggestion
				log.Errorf("Can't reconfigure SDS after having received standard rules: %v", rerr)
```

Reply:
Isn't this exactly the same? Or maybe it'd be the same as `%s`? (and I know `%v` is better in this situation)

Reply:
I was under impression that `...:", rerr)` would print without whitespace, but it doesn't seem to be the case when the logger is initialized. Feel free to ignore.